week4tasks/src/week4/reversing/Main.java

```diff
@@ -0,0 +1,30 @@
+package week4.reversing;
+
+import java.util.Scanner;
+
+class reverser{
+     public reverser revesering(String str){
+          char ch[]= str.toCharArray();
+          String rev= "";
+          for(int i=ch.length-1; i>=0;i--){
+              rev+= ch[i];
+
+          }
+           System.out.print(rev);
+
+
+         return null;
+     }
+
+ }
+
+public class Main {
+
+    public static void main(String[] args) {
+         String S;
```

Comment:
the variables must start with lowercase letters and be meaningful ass there is no mean with S.